python/paddle/fluid/tests/custom_op/utils.py

```diff
@@ -27,7 +37,15 @@
 # PaddlePaddle whl. So here we specific `include_dirs` to avoid errors in CI.
 paddle_includes = [
     os.path.join(site_packages_path, 'paddle', 'include'),
-    os.path.join(site_packages_path, 'paddle', 'include', 'third_party')
+    os.path.join(site_packages_path, 'paddle', 'include', 'third_party'),
+    os.path.join(site_packages_path, 'paddle', 'include', 'third_party',
+                 'eigen3', 'src', 'extern_eigen3'),
+    os.path.join(paddle_path),
+    os.path.join(paddle_path, 'paddle', 'fluid', 'platform'),
+    eigen_path,
+    "C:\\home\\workspace\\cache\\third_party\\cpu\\42f1341df65c796cf2b261c10cd2e4af\\eigen3\\src\\extern_eigen3",
```

Comment:
maybe IS_WINDOWS can be used here.

Reply:
Currently I'm still testing which of these paths are required. I'm adding as many as possible to try to make all the CI's pass. After that I'll remove the unnecessary ones and add IS_WINDOWS checks as you suggest.